Sources/JavaScriptKit/BasicObjects/JSArray.swift

```diff
@@ -43,7 +47,36 @@ extension JSArray: RandomAccessCollection {
 
     public var startIndex: Int { 0 }
 
-    public var endIndex: Int { ref.length.number.map(Int.init) ?? 0 }
+    public var endIndex: Int { count }
```

Comment:
Shouldn't this be `length - 1` since the subscript notation does count holes? Or is this important to using the iterator?

Reply:
Sorry, I accidentally changed that.